### ISSUE TITLE
Source key missing for applications

### ISSUE THREAD

**Issue by oeway** [bioimage-io/collection#95]
Hi, the model download feature on the website is current broken because the `imjoy/BioImageIO-Packager` is generated properly, the `source` key is missing.

For reference, this is how it should look like: https://github.com/bioimage-io/collection-bioimage-io/blob/75e065c30e0f28f579468936ebf10e04dc2216b9/collection.json#L3918-L3935:
```json
{
      "badges": [],
      "covers": [],
      "description": "Exporting BioImage.IO model packages from the model description file",
      "download_count": 1,
      "icon": "https://raw.githubusercontent.com/imjoy-team/bioimage-io-models/master/asset/download-icon.png",
      "id": "imjoy/BioImageIO-Packager",
      "name": "BioImageIO-Packager",
      "rdf_source": "https://bioimage-io.github.io/collection-bioimage-io/rdfs/imjoy/BioImageIO-Packager/latest/rdf.yaml",
      "source": "https://raw.githubusercontent.com/imjoy-team/bioimage-io-models/master/src/bioimageio-packager.imjoy.html",
      "tags": [
        "bioengine"
      ],
      "type": "application",
      "versions": [
        "latest"
      ]
    }
```

This is how it looks like currently in the collection.json file (s3 ebi):
```json
{
            "authors": [],
            "badges": [],
            "covers": [
                "https://github.com/imjoy-team/bioimage-io-resources/blob/master/src/bioengine-icon.svg"
            ],
            "created": "2024-05-17T08:27:52.264437",
            "description": "Exporting BioImage.IO model packages from the model description file",
            "download_count": "?",
            "icon": "https://raw.githubusercontent.com/imjoy-team/bioimage-io-models/master/asset/download-icon.png",
            "id": "imjoy/BioImageIO-Packager",
            "license": "MIT",
            "links": [],
            "name": "BioImageIO-Packager",
            "nickname": "imjoy/BioImageIO-Packager",
            "rdf_sha256": "fa2c40f13cd3df1c872610f12b23ace8c0dd1cf8829970fb468ec6be1a562658",
            "rdf_source": "https://uk1s3.embassy.ebi.ac.uk/public-datasets/bioimage.io/imjoy/BioImageIO-Packager/1/files/rdf.yaml",
            "root_url": "https://uk1s3.embassy.ebi.ac.uk/public-datasets/bioimage.io/imjoy/BioImageIO-Packager/1/files",
            "tags": [
                "bioengine"
            ],
            "type": "application",
            "versions": [
                "1"
            ],
            "dois": [
                null
            ]
        },
```

Basically, this part is missing:
```
"source": "https://raw.githubusercontent.com/imjoy-team/bioimage-io-models/master/src/bioimageio-packager.imjoy.html",
```

cc @FynnBe

**Comment by FynnBe** [comment]
I checked the collection.json generation on the sandbox which included the `source` field... 
Now I just ran the collection.json generation again on production (https://github.com/bioimage-io/collection/actions/runs/11048236485/job/30691137629) and the `source` field is there too...

Not sure why this wasn't the case before. No time to futher investigate this rn though... 

Download via website's download button works now

**Comment by FynnBe** [comment]
last collection.json generation was 2 weeks ago. There were no significnat code changes since then... 
Only thing I can think of that it was (accidentally) overwritten by someone with direct access to S3 (maybe with an older version?) within the last two weeks...

**Comment by oeway** [comment]
Ok, thanks! Not sure if anyone would accidentally produce a wrong collection file, that mostly works but not entirely. Anyway, it's good that the issue is gone now.
